library/core/src/convert/mod.rs

```diff
@@ -370,12 +358,9 @@ pub trait Into<T>: Sized {
 /// }
 /// ```
 ///
-/// [`TryFrom`]: trait.TryFrom.html
-/// [`Option<T>`]: ../../std/option/enum.Option.html
-/// [`Result<T, E>`]: ../../std/result/enum.Result.html
+/// [`Option<T>`]: crate::option::Option
+/// [`Result<T, E>`]: crate::result::Result
 /// [`String`]: ../../std/string/struct.String.html
-/// [`Into`]: trait.Into.html
-/// [`from`]: trait.From.html#tymethod.from
 /// [book]: ../../book/ch09-00-error-handling.html
```

Comment:
`from` isn't a free function.
```suggestion
/// [`from`]: From::from
/// [book]: ../../book/ch09-00-error-handling.html
```